5.5/docker-entrypoint.sh

```diff
@@ -58,6 +58,11 @@ _datadir() {
 	"$@" --verbose --help --log-bin-index="$(mktemp -u)" 2>/dev/null | awk '$1 == "datadir" { print $2; exit }'
 }
 
+# For use with the client if user passes the --socket argument
+_socket() {
+	"$@" --verbose --help --log-bin-index="$(mktemp -u)" 2>/dev/null | awk '$1 == "socket" { print $2; exit }'
+}
```

Comment:
Looks to be working fine. I'll get the pr update up tomorrow once it's applied and tested for all versions.